File: Hello_Node/myinsert.js
var mysql = require('mysql');
var connection = mysql.createConnection({
	host: 'localhost',
	user: 'root',
	password: 'python',
	database: 'python',
	port: 3305
});

connection.connect();

var e_id = '5';
var e_name = '1';
var sex = '1';
var addr = '1';

var sql = `
	INSERT INTO emp 
		(e_id,e_name,sex,addr) 
	VALUES 
		('${e_id}','${e_name}','${sex}','${addr}')
`;

console.log(sql);

connection.query(sql, function (error, result) {
	console.log("-----------------------------------");
	console.log(error);
	console.log("-----------------------------------");
	console.log(result.affectedRows);

});

connection.end();

File: Hello_Python/day14/myinsert.py
import pymysql

conn = pymysql.connect(host='localhost', user='root', password='python', db='python', charset='utf8', port=3305)


e_id = '5'
e_name = '5'
sex = '5'
addr = '5'

cursor = conn.cursor()
sql = f"INSERT INTO emp VALUES ('{e_id}','{e_name}','{sex}','{addr}')"

print(sql)
cnt = cursor.execute(sql)
print(cnt)

conn.commit()
cursor.close()
conn.close()